### ISSUE TITLE
Meeting: OneDrive and SharePoint Photos Next Steps

### ISSUE THREAD

**Issue by dianamartin** [cityofaustin/atd-data-tech#6652]
### Objective 
Review [workflow](https://miro.com/app/board/o9J_l7F_2DE=/) I created comparing OneDrive to MS Teams for photo/video upload

### Participants
Meeting scheduled for 7/14
- Susanne, Christina, Diana

### Agenda
Add agenda here or create agenda from [this template](https://docs.google.com/document/d/1d_49KW5C_vSz8Bs50v-cxyIJuTNJMwMrh7ypcxRHgZI/edit#) and add link. 

------
- [x] Schedule meeting
- [ ] Optional: Schedule debrief
- [ ] Meet and take notes
- [ ] Optional: Debrief with DTS team members
- [ ] Create resulting issues

**Comment by dianamartin** [comment]
We are in agreement that OneDrive will work best for SMB. We need to test creating a new modern SP Document Library if it will show up on the OneDrive App. 

- Then test syncing the new folder to desktop
- Then importing cloud folder to MS Photos
- Need to review MS Photos Slideshow capability

**Comment by dianamartin** [comment]
Susanne said 7/15
>I was able to test sync the folder to desktop. However I ran into issues importing the cloud folder to MS Photos because it required signing in, but the Photos app is restricted for Admin access only.
![image.png](https://images.zenhubusercontent.com/5cb4c11373d7e234297de5cf/0eae98e1-8ef2-4a00-b03f-34456768a70a)

I also looked into Power Automate, but it doesn't have the Photos App as an option either. It might be something that just requires downloading the files locally from OneDrive on an as-needed basis.

For the slideshow feature, I was able to create a slideshow by changing the properties of the folder to "Optimize for pictures" in the OneDrive photo folder in desktop and use the "Picture Tool" tab to full screen a slideshow.
![image.png](https://images.zenhubusercontent.com/5cb4c11373d7e234297de5cf/5bc19d5f-6062-4fa6-90d2-23c48677deca)
